recipes/json-glib/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conans import ConanFile, tools, Meson, VisualStudioBuildEnvironment
+
+import os
+import glob
+import shutil
+
+required_conan_version = ">=1.29"
+
+class JsonGlibConan(ConanFile):
+    name = "json-glib"
+    description = "JSON-GLib implements a full JSON parser and generator using GLib and GObject, and integrates JSON with GLib data types."
+    topics = ("json", "glib")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://wiki.gnome.org/Projects/JsonGlib"
+    license = "GNU LGPL-2.1"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_introspection": ['enabled', 'auto', 'disabled'],
+        "gtk_doc": ['enabled', 'auto', 'disabled'],
```

Comment:
`auto` is a bad option. json-lib is built with `introspection` enabled or disabled, not "auto".
It leads to build-system dependent packages.
```suggestion
        "with_introspection": [True, False],
        "gtk_doc": [True, False],
```

Reply:
I've kept `gtk_doc`, but I doubt this option is needed.
Aren't the docs removed anyways in `package`?
And we try to make lean conan packages, which have the docs removed always.